tests/main/ubuntu-core-create-user/task.yaml

```diff
@@ -7,6 +7,15 @@ restore: |
     sed -i '/^mvo/d' /var/lib/extrausers/group
     rm -rf /home/mvo
 execute: |
+    if [ "$MANAGED_DEVICE" = "true" ]; then
```

Comment:
Yeah, this will use create-user. This specific case is probably okay, as you've necessarily exercised the logic this is supposed to test by creating the testing user already. I wonder if this will create further issues, though, given that we have a dirty snapd state which includes users, etc.